core/src/main/java/com/arcrobotics/ftclib/command/button/Trigger.java

```diff
@@ -283,6 +283,64 @@ public Trigger toggleWhenActive(final Command command) {
         return toggleWhenActive(command, true);
     }
 
+    /**
+     * Toggles between two commands when the trigger becomes active (commadOne then commandTwo
+     * then commandOne).
+     *
+     * @param commandOne the command to toggle
+     * @param commandTwo the command to be toggled
+     * @param interruptible whether the commands are interruptible
+     * @return this trigger, so calls can be chained
+     */
+    public Trigger toggleWhenActive(final Command commandOne, final Command commandTwo, boolean interruptible) {
+        CommandScheduler.getInstance().addButton(new Runnable() {
+            private boolean m_pressedLast = get();
+            @Override
+            public void run() {
+                boolean pressed = get();
+                if (!m_pressedLast && pressed) {
+                    if (commandOne.isScheduled()) {
+                        commandOne.cancel();
+                        commandTwo.schedule(interruptible);
+                    } else if(commandTwo.isScheduled()){
+                        commandTwo.cancel();
+                        commandOne.schedule(interruptible);
+                    } else {
+                        commandOne.schedule(interruptible);
+                    }
+                }
+                m_pressedLast = pressed;
+            }
+        });
```

Comment:
There's a bit of difference in whitespace here that's different from the other methods. Refer to them for the proper spacing.